src/integrationTest/java/uk/gov/hmcts/reform/bulkscan/payment/processor/config/IntegrationTest.java

```diff
@@ -27,5 +27,5 @@
 @AutoConfigureWireMock
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
 @ContextConfiguration(initializers = IntegrationContextInitializer.class)
-@ActiveProfiles({"integration","nosb"}) // no servicebus queue handler registration
+@ActiveProfiles({"integration", "functional"}) // no servicebus queue handler registration
```

Comment:
This is confusing, `@IntegrationTest` annotation activates `functional` profile??